falkordb_bulk_loader/sources.py

```
@@ -0,0 +1,152 @@
+        reader = csv.reader(
+            self._file,
+            delimiter=config.separator,
+            skipinitialspace=True,
+            quoting=config.quoting,
+            escapechar=config.escapechar,
+        )
+
+        try:
+            header = next(reader)
+        except StopIteration:
+            raise CSVError(f"{self.name}: Input file is empty")
```

Ruff (0.15.11)

[warning] Within an `except` clause, raise exceptions with `raise ... from err` or `raise ... from None` to distinguish them from errors in exception handling

(B904)